### ISSUE TITLE
Log spam: Could not locate ordinal parameter [2], expecting one of []

### ISSUE THREAD

**Issue by Brutus5000** [micheljung/faf-java-api#494]
This spams all over the log, making it impossible to spot real errors:

```
2021-08-06 20:45:35.559  INFO 1 --- [                  no request context] [     Executor-5] j.r.q.QueryParameterSetter$ErrorHandling : Silently ignoring   
                                                                                                                                                                                             
java.lang.IllegalArgumentException: Could not locate ordinal parameter [2], expecting one of []                                                                                              
        at org.hibernate.query.internal.ParameterMetadataImpl.getOrdinalParameterDescriptor(ParameterMetadataImpl.java:154)                              
        at org.hibernate.query.internal.ParameterMetadataImpl.getQueryParameter(ParameterMetadataImpl.java:204)                                                                              
        at org.hibernate.query.internal.QueryParameterBindingsImpl.getBinding(QueryParameterBindingsImpl.java:183)                      
        at org.hibernate.query.internal.AbstractProducedQuery.setParameter(AbstractProducedQuery.java:512)                                                  
        at org.hibernate.query.internal.NativeQueryImpl.setParameter(NativeQueryImpl.java:652)                                                                                               
        at org.hibernate.query.internal.NativeQueryImpl.setParameter(NativeQueryImpl.java:65)                                                                                                
        at org.springframework.data.jpa.repository.query.QueryParameterSetter$BindableQuery.setParameter(QueryParameterSetter.java:335)                     
        at org.springframework.data.jpa.repository.query.QueryParameterSetter$NamedOrIndexedQueryParameterSetter.lambda$setParameter$5(QueryParameterSetter.java:128)                        
        at org.springframework.data.jpa.repository.query.QueryParameterSetter$ErrorHandling$2.execute(QueryParameterSetter.java:151)                                                         
        at org.springframework.data.jpa.repository.query.QueryParameterSetter$NamedOrIndexedQueryParameterSetter.setParameter(QueryParameterSetter.java:128)
        at org.springframework.data.jpa.repository.query.ParameterBinder.bind(ParameterBinder.java:82)                                                                                       
        at org.springframework.data.jpa.repository.query.AbstractStringBasedJpaQuery.doCreateCountQuery(AbstractStringBasedJpaQuery.java:122)                                                
        at org.springframework.data.jpa.repository.query.AbstractJpaQuery.createCountQuery(AbstractJpaQuery.java:253)                                       
        at org.springframework.data.jpa.repository.query.JpaQueryExecution$PagedExecution.count(JpaQueryExecution.java:183)                                                                  
        at org.springframework.data.jpa.repository.query.JpaQueryExecution$PagedExecution.lambda$doExecute$0(JpaQueryExecution.java:178)                                                     
        at org.springframework.data.support.PageableExecutionUtils.getPage(PageableExecutionUtils.java:62)                                
        at org.springframework.data.repository.support.PageableExecutionUtils.getPage(PageableExecutionUtils.java:51)                                                                        
        at org.springframework.data.jpa.repository.query.JpaQueryExecution$PagedExecution.doExecute(JpaQueryExecution.java:177)                                                              
        at org.springframework.data.jpa.repository.query.JpaQueryExecution.execute(JpaQueryExecution.java:88)                                               
        at org.springframework.data.jpa.repository.query.AbstractJpaQuery.doExecute(AbstractJpaQuery.java:155)                                                                               
        at org.springframework.data.jpa.repository.query.AbstractJpaQuery.execute(AbstractJpaQuery.java:143)                                                                                 
        at org.springframework.data.repository.core.support.RepositoryMethodInvoker.doInvoke(RepositoryMethodInvoker.java:137)                           
        at org.springframework.data.repository.core.support.RepositoryMethodInvoker.invoke(RepositoryMethodInvoker.java:121)                                                                 
        at org.springframework.data.repository.core.support.QueryExecutorMethodInterceptor.doInvoke(QueryExecutorMethodInterceptor.java:159)
        at org.springframework.data.repository.core.support.QueryExecutorMethodInterceptor.invoke(QueryExecutorMethodInterceptor.java:138)                  
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)                                                                         
        at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:123)                                                       
        at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:388)               
        at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)                                                                        
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)                                                                         
        at org.springframework.dao.support.PersistenceExceptionTranslationInterceptor.invoke(PersistenceExceptionTranslationInterceptor.java:137)           
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)                                                                         
        at org.springframework.data.jpa.repository.support.CrudMethodMetadataPostProcessor$CrudMethodMetadataPopulatingMethodInterceptor.invoke(CrudMethodMetadataPostProcessor.java:145)    
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)                                        
        at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97)                                                                       
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)                                                                         
        at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:215)                                       
        at jdk.proxy2/jdk.proxy2.$Proxy252.getLeaderboardByPage(Unknown Source)                                                                                                              
        at com.faforever.api.leaderboard.LeaderboardService.getGlobalLeaderboard(LeaderboardService.java:35)                                                                                 
        at com.faforever.api.leaderboard.LeaderboardService$$FastClassBySpringCGLIB$$594b252c.invoke(<generated>)                                           
        at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218)                                                                                                          
        at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:779)                                                                     
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)  
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)                                                               [576/1983]
        at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:750)                                                                             
        at org.springframework.cache.interceptor.CacheInterceptor.lambda$invoke$0(CacheInterceptor.java:54)                                                                                  
        at org.springframework.cache.interceptor.CacheAspectSupport.invokeOperation(CacheAspectSupport.java:366)                                            
        at org.springframework.cache.interceptor.CacheAspectSupport.execute(CacheAspectSupport.java:421)                                                                                     
        at org.springframework.cache.interceptor.CacheAspectSupport.execute(CacheAspectSupport.java:345)                                                                                     
        at org.springframework.cache.interceptor.CacheInterceptor.invoke(CacheInterceptor.java:64)                                                       
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)                                                                         
        at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:750)                        
        at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:692)                                      
        at com.faforever.api.leaderboard.LeaderboardService$$EnhancerBySpringCGLIB$$62a156c8.getGlobalLeaderboard(<generated>)                                                               
        at com.faforever.api.leaderboard.LeaderboardController.getGlobal(LeaderboardController.java:61)                                                                                      
        at com.faforever.api.leaderboard.LeaderboardController$$FastClassBySpringCGLIB$$bc885dc5.invoke(<generated>)                                        
        at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218)                                                                                                          
        at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:779)                                                                     
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)                                        
        at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:750)                                                                             
        at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115)                                                                 
        at org.springframework.aop.interceptor.AsyncExecutionAspectSupport.lambda$doSubmit$3(AsyncExecutionAspectSupport.java:276)                          
        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1764)                                                                                     
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)                                                                                         
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)                                      
        at java.base/java.lang.Thread.run(Thread.java:831)                              
```

Is the LeaderboardController still relevant? It should be replaced with the new tmm tables.

**Comment by Sheikah45** [comment]
The only thing still using the leaderboard controller I believe is the website

**Comment by Brutus5000** [comment]
Closed by #499
